kodilla-testing/src/main/java/com/kodilla/testing/forum/ForumUser.java

```diff
@@ -0,0 +1,78 @@
+package com.kodilla.testing.forum;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+
+public class ForumUser {
+    private String name;
+    private String realName;
+    private ArrayList<ForumPost> posts = new ArrayList<ForumPost>();
+    private LinkedList<ForumComment> comments = new LinkedList<ForumComment>();
+
+    public ForumUser(String name, String realName) {
+        // name visible on forum
+        this.name = name;
+        // real name of the user
+        this.realName = realName;
+    }
+
+    public void addPost(String author, String postBody){
+        ForumPost thePost = new ForumPost(postBody, author);
+        posts.add(thePost);
+    }
+
+    public void addComment(ForumPost thePost, String author, String commentBody){
+        ForumComment theComment = new ForumComment(thePost, commentBody, author);
+        comments.add(theComment);
+    }
+
+    public int getPostsQuantity(){
+        return posts.size();
+    }
+
+    public int getCommentsQuantity(){
+        return comments.size();
+    }
+
+    public ForumPost getPost(int postNumber) {
+        ForumPost thePost = null;
+        if(postNumber >= 0 && postNumber < posts.size()){
```

Comment:
Metoda nie powinna nigdy zwracac nulla. Zostaw IndexOutOfBoundsException

Reply:
Ok, ale plik ForumUser.java to przykład robiony w module a nie zadanie :-D Przez przypadek go wcześniej commitowałem... To jest kod napisany przez Kodillę :-D Oczywiście ufam Twoim sugestiom, więc uznaję, że w module są błędne przykłady...